shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -286,7 +286,7 @@ private boolean isSamsung() {
   @Override
   public boolean setSelection(int start, int end) {
     boolean result = super.setSelection(start, end);
-    markDirty();
+    updateEditingState();
```

Comment:
The markDirty() should be called as well before the updateEditingState() to make sure it is not accidentally treated as a repeat call.

Reply:
done